networks/p2p/server.go

```diff
@@ -516,14 +516,11 @@ func (srv *MultiChannelServer) SetupConn(fd net.Conn, flags connFlag, dialDest *
 		c.portOrder = PortOrder(dialDest.PortOrder)
 	} else {
 		for i, addr := range srv.ListenAddrs {
```

Comment:
Is it possible to end without setting `c.portOrder`? What might be happened in that case?

Reply:
no it's not possible. If that happens, it's a bug.